test/logging/example_tlogger_test.go

```diff
@@ -0,0 +1,70 @@
+/*
+Copyright 2020 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package logging_test
+
+import (
+	"testing"
+
+	"knative.dev/pkg/test/logging"
+)
+
+type de struct {
```

Comment:
what's `de` for?

Reply:
Allows the example to compile. It's not important what it is, so I just picked a real short name. Will change to whatever.

Reply:
testStruct? :)

Reply:
done